controllers/admin/AdminSearchConfController.php

```diff
@@ -181,6 +181,49 @@ public function __construct()
                             ),
                         ),
                     ),
+                    'PS_SEARCH_FUZZY' => array(
+                        'title' => $this->trans('Fuzzy Search', array(), 'Admin.Shopparameters.Feature'),
+                        'validation' => 'isBool',
+                        'cast' => 'intval',
+                        'type' => 'bool',
+                        'desc' => $this->trans(
+                                'By default, fuzzy search is enabled and spelling error are allowed. To search “bird”, you can to enter “burd”, “bard”, “beerd” etc...',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ) . '<br/>' .
+                            $this->trans(
+                                'With this option disabled, to get good result, you need get exact writting contained in the word.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                        'hint' => array(
+                            $this->trans(
+                                'Enable spelling errors during search.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                            $this->trans(
+                                'Fuzzy search will check if a close word of the search word exist in indexed words in database. This will be very resource-consuming.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                        ),
+                    ),
+                    'PS_SEARCH_FUZZY_MAX_LOOP' => array(
+                        'title' => $this->trans(
+                            'Maximum words with spelling error allowed by search',
```

Comment:
```suggestion
                            'Maximum approximate strings allowed by search',
```

Reply:
Here, the problem is that we need something really explicite. The user have to understand that this 2 options are linked. Please see attachment : 
![image](https://user-images.githubusercontent.com/6756795/65845420-66a78480-e364-11e9-84d0-35be8174212a.png)

What about : Maximum approximate strings allowed by fuzzy search ? 
(I commit this string for, wait for your opinion)

Reply:
It suits me fine, thanks @Lathanao! Let's keep it as is, with 'approximate strings' and not 'approximated strings'. ;-)